src/webpack/loaderUtils/configHelpers.js

```diff
@@ -14,12 +14,7 @@ const startCase = (string) => {
 /**
```

Comment:
Are there any tests around these ?

Reply:
They're indirectly tested through the generate navigation config tests